### PR TITLE
Release/1.8.0 [5313]

### PR DESCRIPTION
**Pull request by LuisGP**

### CODE REVIEW

File: ShapesDemo.pro [hunk 1]
```diff
@@ -124,3 +121,17 @@ RESOURCES += \
     images/eprosimalogo.qrc
 
 RC_FILE = images/eprosima_icon.rc
+
+win32:CONFIG(release, debug|release): LIBS += -L$$PWD/../../Fast-RTPS/allocation-qos-release/build/install/lib/release/ -lfastrtps
```
Comment (MiguelCompany) [inline ShapesDemo.pro]: These lines seem to be dependent on your build. They shouldn't be necessary, as ShapesDemo is supposed to be built using an installed version of Fast-RTPS. That is why we added the colcon.pgk file

Comment (LuisGP) [reply ShapesDemo.pro]: You are totally right. Removing them.

File: CMakeLists.txt [hunk 2]
```diff
@@ -127,36 +131,54 @@ eprosima_find_package(fastcdr)
...
+
+set( UIS
+    forms/mainwindow.ui
+    forms/optionsdialog.ui
+    forms/publishdialog.ui
+    forms/subscribedialog.ui
+    )
+
+
+#set(CMAKE_AUTORCC ON)
```
Comment (MiguelCompany) [inline CMakeLists.txt]: Are these three commented lines necessary ot not?

File: ShapesDemo.pro [hunk 3]
```diff
@@ -46,8 +46,8 @@ FORMS    +=   forms/mainwindow.ui \
               forms/optionsdialog.ui
 
 
-UI_DIR = $$_PRO_FILE_PWD_/forms/ui/
-MOC_DIR = $$_PRO_FILE_PWD_/forms/ui/
+#UI_DIR = $$_PRO_FILE_PWD_/forms/ui/
```
Comment (MiguelCompany) [inline ShapesDemo.pro]: Remove instead of comment ?

**Pull request merged**